src/main/java/net/minecraft/server/NetLoginHandler.java

```diff
@@ -191,6 +191,18 @@ public void b(Packet1Login packet1login) {
         this.c = true;
     }
 
+    public void a(Packet254GetInfo packet254getinfo) {
```

Comment:
Awaiting changes to prevent exception spam from modern clients querying the server